### DISCUSSION TITLE
can i customize the categories?

### DISCUSSION THREAD

**Discussion by ralyodio** [stackernews/stacker.news#308] [Q&A]
I want to use this software in a different niche.

**Comment by ekzyis** [comment]
What do you mean with categories?

You can definitely fork this repository and do any changes you want since [the license is MIT](https://github.com/stackernews/stacker.news/blob/master/LICENSE)

**Reply by ralyodio** [reply]
like the sub-reddits or whatever they are called.

**Comment by ralyodio** [comment]
is there a bug for this?

**Reply by ekzyis** [reply]
Bug?

**Reply by ralyodio** [reply]
I'm wondering if there is a feature request rather to support custom `/<topics>`

**Comment by ralyodio** [comment]
can we customize price of moderating a territory?

**Reply by ekzyis** [reply]
Can you elaborate?

**Comment by ralyodio** [comment]
like on stacker.news the price is 100k sats/month. I want to change that for default territory prices.
